docs/cody/clients/enable-cody-enterprise.mdx

```diff
@@ -1,17 +1,104 @@
-
-- [Self-hosted Sourcegraph](#cody-on-self-hosted-sourcegraph-enterprise)
-- [Sourcegraph Cloud](#cody-on-sourcegraph-cloud)
+Cody enhances your coding experience by providing intelligent code suggestions, context-aware completions, and advanced code analysis.
 
 <LinkCards>
    <LinkCard href="https://about.sourcegraph.com/cody/pricing" imgSrc="https://sourcegraph.com/.assets/img/sourcegraph-mark.svg" imgAlt="Cody Enterprise" title="Cody Enterprise" description="Get in touch with our team to try Cody for Sourcegraph Enterprise." />
 </LinkCards>
 
-## Cody on self-hosted Sourcegraph enterprise
+## Cody Enterprise features
+
+To cater to your Enterprise requirements, Cody Enterprise offers the following features:
+
+### IDE Token Expiry
+
+<Callout type= "info">IDE Token Expiry is supported with VS Code and JetBrains extensions.</Callout>
+
+Site administrators can customize the time duration of the access token used by other users on the Sourcegraph Enterprise instance to connect Cody from their IDEs via the **Site admin** page. Administrators can choose from various options, including 7, 14, 30, 60, and 90 days.
+
+![ide-token-expiry](https://storage.googleapis.com/sourcegraph-assets/Docs/token-expiry.png)
+
+### Guardrails
+
+<Callout type= "note">Guardrails for public code is currently in Beta stage and is supported with VS Code and JetBrains IDEs extensions.</Callout>
+
+Open source attribution guardrails for public code reduce the exposure to copyrighted code, commonly called copyright guardrails. This involves the implementation of a verification mechanism within Cody to ensure that any code generated by the platform does not replicate open source code.
+
+Guardrails for public code are available to all Sourcegraph Enterprise instances and are **disabled** by default. You can enable it from the **Site configuration** or the IDE extension settings. It only matches code snippets that are at least **10 lines** or longer, and it spans **290,000** search results.
+
+### Admin Controls
+
+<Callout type="note">Admin Controls is supported with VS Code IDE extension.</Callout>
+
+Sourcegraph account admins have selective control over users' access to Cody Enterprise, which is now managed via the Sourcegraph role-based access control system. This provides a more intuitive user interface for assigning permission to use Cody.
+
+Using the access controls, you can also configure which Cody capabilities users can have access to (for example, chat, completion, or commands).
+
+### Analytics
+
+<Callout type= "note">Admin Controls is supported with VS Code IDE extension and on the latest versions of JetBrains IDE.</Callout>
+
+Cody Enterprise users get a clear view of usage analytics for their instance on a self-service basis. A separately managed cloud service for Cody analytics (for example, `cody-analytics.sourcegraph.com`) handles user auth, gets metrics data from Sourcegraph's BigQuery instance, and visualizes the metrics data.
```

Comment:
That's the only prod instance of the Cody Analytics service. I think we can remove the "for example" part.